torch/_torch_docs.py

```diff
@@ -7715,12 +7717,17 @@ def merge_dicts(*dicts):
 
 Example::
 
-    >>> a = torch.randint(10, (5,))
-    >>> a
-    tensor([6, 5, 1, 0, 2])
-    >>> b = a + (torch.randn(50, 1) * 5).long()
+    >>> b = torch.randint(4, (5, 7))
+    >>> b
+    tensor([[0, 0, 0, 2, 0, 0, 2],
+            [0, 3, 0, 0, 2, 0, 1],
+            [2, 2, 2, 0, 0, 0, 3],
+            [2, 2, 3, 0, 1, 1, 0],
+            [1, 1, 0, 0, 2, 0, 2]])
```

Comment:
Make the example more easily reproducible:

```suggestion
    >>> b = torch.tensor(
           [[0, 0, 0, 2, 0, 0, 2],
            [0, 3, 0, 0, 2, 0, 1],
            [2, 2, 2, 0, 0, 0, 3],
            [2, 2, 3, 0, 1, 1, 0],
            [1, 1, 0, 0, 2, 0, 2]])
```